modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -160,7 +162,7 @@ private void doStartNode(final String defaultConfig) {
         }
 
         int dev = 0;
-        if (this.debug) {
+        if (this.devCluster) {
```

Comment:
Why this change? I would imagine debug => devCluster, not other way around

Reply:
`BuildConfig.DEBUG` is more about development and prod mode of application, when this `devCluster` means that devCluster can be used even if app wasn't compiled for local development.